File: Aide.Admin.Domain/Objects/InsuranceExportProbatoryDocument.cs
using Aide.Admin.Domain.Enumerations;
using System;

namespace Aide.Admin.Domain.Objects
{
    public class InsuranceExportProbatoryDocument
    {
        public int Id { get; set; }
        public EnumExportTypeId ExportTypeId { get; set; }
        public int InsuranceCompanyId { get; set; }
        public EnumClaimTypeId ClaimTypeId { get; set; }
        public EnumExportDocumentTypeId ExportDocumentTypeId { get; set; }
        public int SortPriority { get; set; }
        public int? ProbatoryDocumentId { get; set; }
        public int? CollageId { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }

        public string Name { get; set; }
    }
}

File: Aide.ClientApp/src/app/models/insurance-export-probatory-document.ts
import { ClaimTypeId } from '../enums/claim-type-id.enum';
import { ExportDocumentTypeId } from '../enums/export-document-type-id.enum';
import { ExportTypeId } from '../enums/export-type-id';

export class InsuranceExportProbatoryDocument {
    exportTypeId: ExportTypeId;
    insuranceCompanyId: number;
    claimTypeId: ClaimTypeId;
    exportDocumentTypeId: ExportDocumentTypeId;
    sortPriority: number;
    probatoryDocumentId?: number;
    collageId?: number;
    name: string;
    dateCreated?: string;
    dateModified?: string;
}
